server/routes/shared_api/place.py

```diff
@@ -257,15 +257,15 @@ def api_i18n_name():
 def get_named_typed_place():
   """Returns data for NamedTypedPlace, a dictionary of key -> NamedTypedPlace."""
   dcids = request.args.getlist('dcids')
-  place_types = get_place_types(dcids)
+  place_type = get_place_type(dcids)
```

Comment:
supernit: I prefer the name`place_types` (plural) because this object holds types for multiple dcids

Reply:
changed to place2type.